### ISSUE TITLE
Highlight the Market Fee Reward better

### ISSUE THREAD

**Issue by startailcoon** [bitshares/bitshares-ui#2966]
With the introduction of the market fee reward there have been numerous assets taking advantage of this type of leverage to attract volume. However, the UI does not take advantage of this to guide the user to assets with high rewards. It's only visible on the asset details page.

We should find a good strategy to display this when users search for assets to trade. There are several places, while some may be difficult and are part of other suggestions for improvements as well.

1. The assets page under explorer as another column. Low impact and good visibility. 
2. The "main page" with assets as another column. Low impact and good visibility.
3. The "exchange page". In the Buy/Sell form we should highlight the market_fee_reward return somewhere. Risks are small spaces and to much information.
4. The "my markets" list. As this space is small, we could add a "rewards" sign to assets in the list, to signal it has a market_fee_reward. A tooltip could display the actual value.

These are the spots I can think of now. Ideas or suggestions?

**Comment by sschiessl-bcp** [comment]
Only the reward is not enough information there. We could add something like effective market fee which takes it into account.

At the end the user wants to know how much it pays per trade in percent fee

**Comment by startailcoon** [comment]
True, a market with 80% return with a 1% fee is less good than a 20% return and 0.1% fee.

**Comment by abitmore** [comment]
IMHO it's not a good idea to show the detailed info to normal users E.G. how their paid market fees would be split to whom at what percentages and why, because it will reduce their desire to trade thus will harm liquidity and volume.

Please consider showing different UI to LTMs and normal users.

Actually, I think it's as simple as showing a tip to LTMs "x% of fees paid will be refunded since you're a LTM, please check and withdraw them in vesting balances".

For normal users, we can show a tip "if you upgrade to LTM, x% of fees paid will be refunded to you".

**Comment by startailcoon** [comment]
> IMHO it's not a good idea to show the detailed info to normal users E.G. how their paid market fees would be split to whom at what percentages and why, because it will reduce their desire to trade thus will harm liquidity and volume.

It's not about showing to where the fee or fee reward goes. This should just view the current fee and reward share in more places than the current asset page. We shouldn't view anything like "x% of your fee goes to user a and user b", that is not relevant. 

> Please consider showing different UI to LTMs and normal users.

It would only be a small difference in the wording, and for normal users it would be more about "if you upgrade" notice. Having the fee percentage and reward share visible in the list of assets for example, wouldn't be any different from a normal user and a LTM.
